wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/consentprompt/ConsentPromptData.kt

```diff
@@ -0,0 +1,38 @@
+package com.android.identity_credential.wallet.ui.destination.consentprompt
+
+import android.graphics.Bitmap
+import androidx.compose.runtime.Immutable
+import androidx.compose.runtime.Stable
+import com.android.identity.credential.CredentialRequest
+import com.android.identity.trustmanagement.TrustPoint
+
+/**
+ * ConsentPromptData is passed to ConsentPrompt to show the appropriate requested credentials.
+ */
+@Stable
+@Immutable
+data class ConsentPromptData(
+    // Object extracted from the document request
+    val credentialRequest: CredentialRequest,
+    // requested doc type
+    val docType: String,
+    // document name of credential being used to respond with requested data
+    val documentName: String,
+    // id of credential that provides documentName - used after Consent Prompt succeeds
+    val credentialId: String,
+    // party requesting to verify user's data
+    val verifier: TrustPoint? = null,
+    // if verifier TrustPoint is present with an icon byte array, process the bytes to bitmap once
+    val verifierIconBitmap: Bitmap? = null
```

Comment:
Since we are referencing the icon in a composable, then compose would have to compute a bitmap from bytes with every recomposition -- this produces unnecessary heavy processing. Composable functions should ideally receive bitmaps as parameters and so processing bytes to bitmap outside of the composable makes sense, otherwise, we may have to use a side-effect to launch a 1-time computation that updates the composable state with a bitmap (ideally composables are side-effect-free).

Reply:
This has a premature optimization smell to it. Maybe just add `// TODO: we're computing a bitmap every composition and this could be slow` comment for now?